tests/test_kwargs.py

```diff
@@ -0,0 +1,166 @@
+from unittest import TestCase
+import numpy as np
+import dianna
+from tests.methods.test_onnx_runner import generate_data
+from tests.utils import load_movie_review_model
+from tests.utils import run_model
+
+
+class ImageKwargs(TestCase):
+    """Suite of tests for kwargs to explainers for Images."""
+
+    def test_lime_image_correct_kwargs(self):
+        """Test to ensure correct kwargs to lime run without issues."""
+        model_filename = 'tests/test_data/mnist_model.onnx'
+        input_data = generate_data(batch_size=1)[0].astype(np.float32)
+        axis_labels = ('channels', 'y', 'x')
+        labels = [1]
+
+        dianna.explain_image(model_filename,
+                                input_data,
+                                method='LIME',
+                                labels=labels,
+                                kernel=None,
+                                kernel_width=25,
+                                verbose=False,
+                                feature_selection='auto',
+                                random_state=None,
+                                axis_labels=axis_labels,
+                                preprocess_function=None,
+                                top_labels=None,
+                                num_features=10,
+                                num_samples=10,
+                                return_masks=True,
+                                positive_only=False,
+                                hide_rest=True,
+                                )
+
+    def test_lime_image_extra_kwarg(self):
+        """Test to ensure extra kwargs to lime raise warnings."""
+        model_filename = 'tests/test_data/mnist_model.onnx'
+        input_data = generate_data(batch_size=1)[0].astype(np.float32)
+        axis_labels = ('channels', 'y', 'x')
+        labels = [1]
+
+        with self.assertWarns(Warning):
+            dianna.explain_image(model_filename,
+                                    input_data,
+                                    method='LIME',
+                                    labels=labels,
+                                    kernel=None,
+                                    kernel_width=25,
+                                    verbose=False,
+                                    feature_selection='auto',
+                                    random_state=None,
+                                    axis_labels=axis_labels,
+                                    preprocess_function=None,
+                                    top_labels=None,
+                                    num_features=10,
+                                    num_samples=10,
+                                    return_masks=True,
+                                    positive_only=False,
+                                    hide_rest=True,
+                                    extra_kwarg=None
+                                    )
```

Comment:
can you also add a test to check the message of warning specifically `{kwargs}` is correct?

Reply:
Combined the two with assertWarnsRegex in the latest commit.